modules/ding_library/ding_library.strongarm.inc

```diff
@@ -171,7 +215,7 @@ function ding_library_strongarm() {
   $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
   $strongarm->api_version = 1;
   $strongarm->name = 'language_content_type_ding_library';
-  $strongarm->value = '0';
+  $strongarm->value = '2';
```

Comment:
What does this change?

Reply:
This shows that Multilingual support is enabled for this content type.
0 - Disabled
1 - Enabled
2 - Enabled with translation